packages/shared/styles/components/SfArrow.scss

```diff
@@ -9,6 +9,13 @@ $arrow__icon-color--hover: $c-dark !default;
 
 .sf-arrow {
   $this: &;
+  &__icon {
+    position: relative;
+    flex: 0 0 24px;
+    height: 12px;
```

Comment:
Shouldn't we have variables for this?

Reply:
Fixed in https://github.com/DivanteLtd/storefront-ui/pull/450/commits/c1385a3e3395189441fac8b7734bf2ea13bba3e4